feature-detects/css/cssgrid.js

```diff
@@ -0,0 +1,19 @@
+/*!
+{
+  "name": "CSS Grid (old & new)",
+  "property": ["cssgrid", "cssgridlegacy"],
+  "tags": ["css"],
+  "notes": [{
+    "name": "The new, standardized CSS Grid",
+    "href": "https://www.w3.org/TR/css3-grid-layout/"
+  }, {
+    "name": "The _old_ CSS Grid",
+    "href": "https://www.w3.org/TR/2011/WD-css3-grid-layout-20110407/"
+  }],
+  "warnings": ["`grid-columns` is only in the old syntax, `grid-column` exists in both and so `grid-template-rows` is used for the new syntax."]
```

Comment:
this isn't really a warning. Warnings should be about the detect themselves (i.e. 🚨 giamundo test🚨 , or listing possible edge case failures. Since the end user of modernizr won't need this information, but modernizr developers would, could you move this to a straight up JS comment on line 17?